packages/openapi-v3-types/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@loopback/openapi-v3-types",
+  "version": "1.0.0-alpha.1",
+  "description": "TypeScript type definitions for OpenAPI Specifications.",
+  "engines": {
+    "node": ">=8"
+  },
+  "dependencies": {
+    "openapi3-ts": "^0.7.0"
+  },
+  "devDependencies": {
+    "@loopback/build": "^4.0.0-alpha.13"
+  },
+  "scripts": {
+    "build": "lb-tsc es2017",
+    "build:apidocs": "lb-apidocs",
+    "clean": "lb-clean loopback-openapi-v3-types*.tgz dist package api-docs",
+    "prepublishOnly": "npm run build && npm run build:apidocs",
+    "verify": "npm pack && tar xf loopback-v3-types*.tgz && tree package && npm run clean"
```

Comment:
`loopback-openapi-v3-types` I think

Reply:
It's a bit worrisome that these comments are pointing out mismatches between scripts and filenames, but the tests are all passing.

Reply:
The easiest way to see for yourself is to hop into the package and run `npm pack` to see what it produces. In this case, it should be `loopback-openapi-v3-types*.tgz`

Reply:
correct, it's `loopback-openapi-v3-types-0.1.0.tgz` so I think the convention I post above is correct, `loopback-` + `<package name>`